image-builder/Dockerfile

```diff
@@ -0,0 +1,3 @@
+From fedora:27
```

Comment:
Can this value come from a top-level `./versions.txt` to make it simpler to detect when this file needs to be updated?

Reply:
I am sorry but, Dockeriles does not support source from another file, (I can point to latests ?  and pray will work always)

Reply:
I guess you could use `latest`, or you could create `Dockerfile.in` so that it could then specify:

```
From: fedora:@fedora_docker_version@
```
or even,

```
From: @docker_distro_name@:@docker_distro_version@
```

... and do `sed` magic to translate variables from `versions.txt` when we "build" the `Dockerfile`.